### ISSUE TITLE
Unable to clone repo with git lfs after 2.2.0 update

### ISSUE THREAD

**Issue by inkychris** [git-lfs/git-lfs#2366]
## Behaviour

As of the 2.2.0 update, calling `git lfs clone` returns the following:
```
error: unknown option `jobs'
usage: git clone [<options>] [--] <repo> [<dir>]

    ...
```
Calling the same command with 2.1.1 returns the expected result:
```
You must specify a repository to clone.

usage: git clone [<options>] [--] <repo> [<dir>]

    ...
```

## Environment
Both versions were tested inside a docker container using the following dockerfile (the `apt-get install git-lfs=VERSION`) being changed accordingly:
```
FROM ubuntu:16.04

RUN apt-get update
RUN apt-get install curl git -y

RUN curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | bash
RUN apt-get install git-lfs=2.2.0 -y

RUN git lfs install
```
Calling:
```
> docker build . -t git-lfs-test
> docker run --rm git-lfs-test git lfs clone
```
Also tried ubuntu:14.04 base image with same result.

Alternatively, without docker I was able to clone again with `git lfs clone` after installing 2.1.1 again. Just wanted to make sure it wasn't an environment issue before submitting an issue.

**Comment by eaweller** [comment]
I have similar results from git lfs clone on a fresh ubuntu 14.04 virtual machine.

git version 1.9.1
git-lfs/2.2.0

git clone (no lfs) provides a workaround, but slow.

**Comment by technoweenie** [comment]
Ah, sorry about that! I posted a fix: https://github.com/git-lfs/git-lfs/pull/2369. I'll have to look into why our git tests on older versions didn't catch this. Should be able to roll v2.2.1 out next week.

You can work around it by disabling LFS during the `clone`:

```sh
$ GIT_LFS_SKIP_SMUDGE=1 git clone some-repo
$ cd some-repo
$ git lfs pull
```

**Comment by Permafacture** [comment]
When might the debian package on packagecloud.io be updated?  Following the instructions on the wiki leads to getting a build with this bug in it. https://github.com/git-lfs/git-lfs/wiki/Installation

**Comment by technoweenie** [comment]
v2.2.1 hasn't been released yet due to some travel time in the team, and a US holiday. @ttaylorr and I just talked about releasing v2.2.1 on Monday, 10 July, so we can incorporate fixes for other bugs from v2.2.0 too.
